inmap/configExample.toml

```diff
@@ -127,6 +127,47 @@ CensusPopColumns= ["TotalPop", "WhiteNoLat", "Black", "Native", "Asian", "Latino
 # in CensusPopColumns.
 PopGridColumn= "TotalPop"
 
+# ConstantsFile is the path to the CSV containing population constants.
+ConstantsFile= "${GOPATH}/src/github.com/spatialmodel/inmap/inmap/testdata/popConstants.csv"
+
+# # ConstantsColumns is a list of the data fields in ConstantsFile that
+# # contain population constants.
+ConstantsColumns= [
```

Comment:
We should be able to extract this information automatically from the file, right?